gameDataFetch.js

```diff
@@ -0,0 +1,34 @@
+const request = require('request-promise');
+const IgdbApiWrapper = require('./IgdbApiWrapper');
+
+let IgdbWrapper = new IgdbApiWrapper();
```

Comment:
Please use const instead of let here.
```suggestion
const IgdbWrapper = new IgdbApiWrapper();
```